scripts/ccpp_state_machine.py

```diff
@@ -3,6 +3,7 @@
 # CCPP framework imports
 from state_machine import StateMachine
 
+_REG_ST = r"(?:register)"
```

Comment:
Do we want to allow truncated register phase names (i.e. `_REG_ST = r"(?:reg(?:ister)?)")`?

Reply:
Would be good if we could avoid that imo. If we hit the 64-character limit for the auto-generated subroutines, we may have to reconsider.